### ISSUE TITLE
Basic deployment is not accessible due to firewall permissions

### ISSUE THREAD

**Issue by tiagoshibata** [Azure/pcs-cli#223]
# Type of issue
- [X] Bug
- [ ] New feature
- [ ] Enhancement

# Description
Basic deployment creates the AD and resources. The CLI then fails to check the service status and shows me

```
Website not yet available, please refer to troubleshooting guide here:
https://aka.ms/iot-rm-tsg
```

Accessing the address gives me a page with:

```
An error occurred while sending the request. - The operation timed out
```

Using SSH to the machine and accessing the page works. I found that the access is blocked in the network security group. HTTP and HTTPS should be added to the allowed firewall rules.

SSH is also enabled to ease debugging.

# Steps to reproduce

1. Deploy with `-t remotemonitoring -s basic -e AzureCloud -r dotnet`.
2. All resources will be deployed, but the CLI will fail to check the service status because of the blocked port. `Website not yet available, please refer to troubleshooting guide here` will be shown on the terminal.
3. Accessing the portal through a browser fails.

# Known workarounds

Adding the rule manually.

# Possible solution

Changing the deployment configuration to allow HTTP and HTTPS. I'm working on a PR and will open it today.

**Comment by timlaverty** [comment]
@tiagoshibata What version of CLI and containers are you deploying?  i.e. the CLI version from github that deploys testing containers or the released PublicPreview.3 containers installed using the nodejs.org version of GitHub?

**Comment by tiagoshibata** [comment]
Latest master (was testing on 6700f8adbc99a045eb7e4b14b05636534b6bb12d).

**Comment by tiagoshibata** [comment]
Fixed by #226.
